docs/spark/tutorials/ml-sentment-analysis.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Sentiment analysis with .NET for Apache Spark and ML.NET tutorial
+description: In this tutorial, you learn how to use ML.NET with .NET for Apache Spark for sentiment analysis.
+author: mamccrea
+ms.author: mamccrea
+ms.date: 03/07/2019
+ms.topic: tutorial
+---
+
+# Tutorial: Sentiment analysis with .NET for Apache Spark and ML.NET
+
+This tutorial teaches you how to perform sentiment analysis of online reviews using ML.NET and .NET for Apache Spark. ML.NET is a free, cross-platform, open-source machine learning framework. You can use ML.NET with .NET for Apache Spark to scale the training and prediction of machine learning algorithms.
+
+In this tutorial, you learn how to:
+
+> [!div class="checklist"]
+>
+> * Write a .NET for Apache Spark app that performs sentiment analysis.
+> * Use the ML.NET model builder in Visual Studio.
+> * 
+
+## Prerequisites
+
+If this is your first .NET for Apache Spark application, start with the [Getting Started tutorial](get-started.md) to become familiar with the basics.
+
+This tutorial uses the ML.NET Model Builder (preview), a visual interface available in Visual Studio. If you do not already have Visual Studio, you can [download the Community version of Visual Studio](https://visualstudio.microsoft.com/downloads/) for free.
+
+[Download and install](https://marketplace.visualstudio.com/items?itemName=MLNET.07) ML.NET Model Builder (preview).
+
```

Comment:
Should we provide any sort of docs links to learn more about Sentiment Analysis and/or ML.NET?